INSTALL.md

```diff
@@ -21,6 +21,17 @@ sudo apt-get update && sudo apt-get install -y \
     libpcap-dev meson pkg-config
 ```
 
+### Hyperscan support
+
+The cndpfwd example application supports a simple hyperscan forwarding type mode. The Hyperscan
```

Comment:
HS used with cndpfwd is pretty much supporting a pattern matching permit/deny list?

Reply:
I was thinking HS can provide a simple DDOS or DPI or even a load balancer, but this is far from a complete application for one of these types.